server/chat-plugins/username-prefixes.ts

```diff
@@ -109,11 +153,14 @@ export const commands: Chat.ChatCommands = {
 
 			const types = target ? [prefixManager.validateType(toID(target))] : ['privacy', 'modchat'];
 
+			const entries = Config.forcedprefixes.filter((x: any) => types.includes(x.type));
+
 			return this.sendReplyBox(types.map(type => {
-				const info = Config.forcedprefixes[type].length ?
-					`<code>${Config.forcedprefixes[type].join('</code>, <code>')}</code>` : `none`;
+				const prefixes = entries.filter((x: any) => x.type === type).map((x: any) => x.prefix);
+				const info = prefixes.length ?
+					`<code>${prefixes.join('</code>, <code>')}</code>` : `none`;
 				return `Username prefixes that disable <strong>${type}</strong>: ${info}.`;
-			}).join(`<br />`));
+			}).join('<br />'));
```

Comment:
Why change this? PS' string guidelines stipulate that `` ` `` should be used for HTML. IMO it doesn't matter enough to change code that already misuses it, but why go out of your way to change code that already adheres to our quote conventions?

Reply:
It wasn't intentional... I accidentally converted it because i deleted the code then rewrote it from memory